package.json

```diff
@@ -23,7 +23,7 @@
 	"dependencies": {
 		"axios": "^0.21.1",
 		"json5": "^2.2.0",
-		"meow": "^10.1.1",
+		"unix-normalization": "^1.0.0",
```

Comment:
we shouldn't need this dependency anymore since we included the whole algorithm in `s/aspects/utilities.ts`

will drop this change